observability/logging/logging-6.0/log6x-loki.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-loki"]
+= Storing logs with LokiStack
+include::_attributes/common-attributes.adoc[]
+:context: logging-6x
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrastructure-related logs.
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You have installed the {loki-op} by using the CLI or web console.
+* You have a `serviceAccount` in the same namespace in which you create the `ClusterLogForwarder`.
+* The `serviceAccount` is assigned `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles.
+
+include::modules/log6x-loki-rbac-rules-perms.adoc[leveloffset=+1]
+
+include::modules/log6x-enabling-loki-alerts.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-memberlist-ip.adoc[leveloffset=+1]
+
+include::modules/log6x-identity-federation.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-pod-placement.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-reliability-hardening.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-restart-hardening.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-retention.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-zone-aware-rep.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-zone-fail-recovery.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-rate-limit-errors.adoc[leveloffset=+1]
```

Comment:
Does it make sense to provide a bit more structure to the topics in this file? Currently it looks as if all the information would be necessary for all usecases and there's also no "this is what you need to do" kind of instruction. I think I would structure it as "this is the minimal configuration", "this is what we recommend you look at" and a "these features are advanced and might not be needed in most cases".